apps/studio/src/schemas/page.ts

```diff
@@ -1,8 +1,10 @@
-import { ISOMER_PAGE_LAYOUTS } from "@opengovsg/isomer-components"
 import { z } from "zod"
 
-const LAYOUT_VALUES = [...Object.values(ISOMER_PAGE_LAYOUTS)]
-type LayoutValues = (typeof LAYOUT_VALUES)[number]
+const NEW_PAGE_LAYOUT_VALUES = [
+  "article",
+  "content",
+] satisfies PrismaJson.BlobJsonContent["layout"][]
```

Comment:
```suggestion
] as const satisfies PrismaJson.BlobJsonContent["layout"][]
```

Reply:
if you add const it doesn't satisfy. cause the output is just a subset.